tasks/tests/unit/test_test_results_processor_task.py

```diff
@@ -20,6 +20,78 @@
 
 
 class TestUploadTestProcessorTask(object):
+    def test_compute_name(self):
+        assert (
+            TestResultsProcessorTask().compute_name(
```

Comment:
as a followup, it would be nice to move this method out of `TestResultsProcessorTask`

Reply:
oh i think that second commit wasn't actually supposed to make it in to this PR, i will fix that